### PR TITLE
sideloadly.sideloadly version 0.25.0

### PR DESCRIPTION
**Pull request by Mudb0y**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56755)

### CODE REVIEW

File: manifests/s/sideloadly/sideloadly/0.25.0/sideloadly.sideloadly.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: sideloadly.sideloadly
+PackageVersion: 0.25.0
+PackageLocale: en-US
+Publisher: Sideloadly
+PackageName: Sideloadly
+License: Unknown
```
Comment (Trenly) [inline manifests/s/sideloadly/sideloadly/0.25.0/sideloadly.sideloadly.locale.en-US.yaml:9]: If the license is not known, assume it is proprietary
```suggestion
License: Proprietary
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — COMMENTED** [review]
Does this program not show up in Add or Remove Programs?

**Comment by stas-prze** [comment]
> > Does this program not show up in Add or Remove Programs?
> 
> Last time I checked it doesn't.
> 
> Also this should be moved to https://github.com/microsoft/winget-pkgs/tree/master/manifests/i/iOSGods/Sideloady.

The above mentioned version is unofficial to my knowledge.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> The above mentioned version is unofficial to my knowledge.

iOSGods owns and are the creators of Sideloadly (all community donations go to them) so you should be able to move it via Visual Studio Online.

**Comment by itsmeboy0** [comment]
@ItzLevvie What's the reason that package is called `Sideloady` instead of `Sideloadly`? Might want to rename that.

**Comment by OfficialEsco** [comment]
Unexpected typo because of no ARP entries 👀

**Pull request closed without merging**